wsmaster/che-core-api-workspace/src/main/resources/schema/devfile.json

```diff
@@ -174,8 +192,21 @@
                     "description": "Describes the component id. It has the following format: [{REGISTRY_URL}/]{plugin/editor PUBLISHER}/{plugin/editor NAME}/{plugin/editor VERSION}, where {REGISTRY_URL}/ is an optional part.",
                     "pattern": "^((https?://)[a-zA-Z0-9_\\-./]+/)?[a-z0-9_\\-.]+/[a-z0-9_\\-.]+/[a-z0-9_\\-.]+$",
```

Comment:
Looks like pattern and description should be updated.
Do not forget to actualize new cases with tests.

Reply:
\+